ShowerShow/DTO/ShowerThoughtDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShowerShow.DTO
+{
+    public class ShowerThoughtDTO
```

Comment:
Frontend had a design where shower thoughts could get favorited. I dont know if I remember it right can you check if they need any more variables?